frontend/packages/ceph-storage-plugin/src/components/dashboards/object-service/storage-efficiency-card/storage-efficiency-card.tsx

```diff
@@ -95,7 +95,7 @@ const StorageEfficiencyCard: React.FC<DashboardItemProps> = ({
       !!savingsQueryResultError || !!logicalSavingsQueryResultError || !savings || !logicalSize,
     title: t('ceph-storage-plugin~Savings'),
     infoText: t(
-      'ceph-storage-plugin~Savings shows the uncompressed and non-deduped data that would have been stored without those techniques.',
+      'ceph-storage-plugin~The Savings metric represents the actual disk capacity saved inclusive of all compression-enabled pools and associated replicas.',
```

Comment:
We dont need to change in object service card.

Reply:
ACK